server_admin/topics/users/attributes.adoc

```diff
@@ -1,11 +1,18 @@
+[configuring-user-attributes]
+= Configuring user attributes
 
-=== User Attributes
+In addition to adding basic user metadata, such as names and email, you can also store arbitrary user attributes.  
 
-Beyond basic user metadata like name and email, you can store arbitrary user attributes.  Choose a user to manage
-then click on the `Attributes` tab.
+.Prerequisite
```

Comment:
The guideline is Prerequisites (plural) even if there is only one.

Reply:
I think this was pointed out at a minimalism session. It should be 'prerequisite'  if there is only one bullet point.

Reply:
Okay, good. that makes more sense.  I will assume that I was told incorrectly.